torch/autograd/profiler.py

```diff
@@ -97,6 +133,10 @@ class profile(object):
         enabled (bool, optional): Setting this to False makes this context manager a no-op.
             Default: ``True``.
 
+        use_cuda (bool, optional): Enables timing of CUDA events as well using the cudaEvent API.
+            Adds approximately 4us of overhead to each tensor operation.
```

Comment:
It is probably about the same for recording timing, but it gives us more control over what we time. E.g. we won't time all individual kernels or cuda API calls, so I expect the overhead will be a bit lower. Really though this is to make it possible to get traces with GPU numbers without needing to export/import them.

Reply:
I didn't mean to say that we shouldn't merge this because of overhead. It's just that if you consider that most of our ATen calls take 1-4us, then extra 4us really is a lot.

Reply:
Yeah, this is for debugging convnet performance, so those overheads are more acceptable.

Reply:
Sure. That still makes sense in some cases

Reply:
BTW 4us is a bit abstract for most people, except for core devs who know how much 4us is. Maybe at least add a bit of context (mention avg. cost of autograd operation)?